src/app/books/books.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { BooksService } from './books.service';
+
+@Component({
+  selector: 'app-books',
+  templateUrl: './books.component.html',
+  styleUrls: ['./books.component.scss'],
+})
+export class BooksComponent implements OnInit {
+  readonly books$ = this.booksService.getBooks();
```

Comment:
Type books$ and why is it _readonly_?

Reply:
Also I don't like that you have called this function here. Better solution is to call it in ngOnInit()